docs/tutorials/faq.md

```diff
@@ -233,7 +233,7 @@ an HLS manifest, we do our best to guess what the codecs might be, but those
 guesses might not always be accurate.  If an HLS manifest has no codec
 information provided, we default to guessing that the video codec is
 `avc1.42E01E` and the audio codec is `mp4a.40.2`, which can cause problems if
-the stream is actually video-only or audio-only.  In this case, you can enable
+the stream is video-only or audio-only.  In this case, you can enable
```

Comment:
I think this should be reverted, too.  The emphasis of "actually" is meaningful, to show the distinction between what we guessed, and reality.  "We thought it was this, but it was _actually_ that."

Reply:
We are clearly stating that the stream is __video-only__ or __audio-only__. There's a clear cut distinction, and the sentence is explicit even without *"actually"* because we used "__only__" which it's putting direct emphasis, thus, making "actually" redundant.

Although I don't quite agree with you on this, I can revert the changes if you say so.